Lombiq.HelpfulExtensions.Tests.UI/Constants/WidgetTypes.cs

```diff
@@ -5,6 +5,6 @@ public static class WidgetTypes
     public const string Container = nameof(Container);
     public const string Html = nameof(Html);
     public const string Liquid = nameof(Liquid);
-    public const string Menu = nameof(Menu);
+    public const string Menu = "Menu Navigation Provider";
```

Comment:
This is a technical name and should rather stay (and definitely not contain spaces).

Reply:
Ah no, that's the _other_ `WidgetTypes`.